src/deque.rs

```diff
@@ -1440,4 +1484,22 @@ mod tests {
         q.pop_front().unwrap();
         q.swap(0, 2);
     }
+
+    #[test]
+    fn try_from_array() {
+        assert!(Deque::<u8, 3>::try_from([1, 2, 3, 4]).is_err());
+
+        let deq1 = Deque::<u8, 8>::try_from([1, 2, 3, 4]).unwrap();
+        let mut deq2 = Deque::<u8, 8>::new();
+        deq2.push_back(1).unwrap();
+        deq2.push_back(2).unwrap();
+        deq2.push_back(3).unwrap();
+        deq2.push_back(4).unwrap();
+
+        // todo change to `assert_eq!(deq1, deq2);` when PR #521 is merged.
+        assert_eq!(deq1.len(), deq2.len());
```

Comment:
```suggestion
        assert_eq!(deq1, deq2);
```

#521 was merged last week :tada: